commands/iam.js

```diff
@@ -13,34 +15,63 @@ const validateEmail = (email) => {
       );
 };
 
-const iam = async (email, user_id) => {
-    if (!validateEmail(email)) {
-        return [null, 'Your email doesn\'t appear to be a valid email. Please try again'];
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('iam')
+		.setDescription('Set your email address with the server')
+		.addStringOption((option) =>
+			option
+				.setName('email')
+				.setDescription('Your UCLA email address (or whatever email you think we have in our directory)')
+				.setRequired(true),
+		),
+    async execute(interaction) {
```

Comment:
let's also define actual functions instead of just inline in the `module.exports`. It helps a lot with readability.

For example, we should have 
```js
async function execute(interaction) {
// function body
}

module.exports = {
    data: // data,
    execute
}
```